content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -157,7 +157,7 @@ accompanying Istio proxy sidecar by checking that the pod meets all of the follo
 
 To redirect traffic in the application pod's network namespace to/from the Istio proxy sidecar, the Istio
 CNI plugin configures the namespace's iptables.  The following table describes the parameters to the
-redirect functionality.  The default values for the parameters are able to be overridden by setting the
+redirect functionality.  The default values for the parameters can be overridden by setting the
```

Comment:
It might be worth mentioning that these parameters can also be set via helm values.

Reply:
Done.